src/internal/kopia/wrapper.go

```diff
@@ -451,6 +452,11 @@ func (w Wrapper) ProduceRestoreCollections(
 		}
 
 		c.streams = append(c.streams, ds)
+
+		loadCount++
+		if loadCount%1000 == 0 {
+			logger.Ctx(ctx).Infow("loading items from kopia", "loaded_count", loadCount)
```

Comment:
This looks good to me. Did you also consider sampling loadCount every x seconds or so?

Reply:
Should we also add another log which indicates that we finished loading all items from kopia?

Reply:
>Did you also consider sampling loadCount every x seconds or so?

we probably don't need to do sampling yet. Mostly we're trying to make sure we keep making some progress as a debugging aid. The timestamps in the log can also be used to calculate rate if we need it

>Should we also add another log which indicates that we finished loading all items from kopia?

good idea, I've added a log for that